Can't compile it.strftime with lcd display

<!-- Thanks for reporting a bug for this project. READ THIS FIRST:
- Please make sure to submit issues in the right GitHub repository, if unsure just post it here:
  - esphomeyaml [here] - This is mostly for reporting bugs when compiling and when you get a long stack trace while compiling or if a configuration fails to validate.
  - esphomelib [https://github.com/OttoWinter/esphomelib] - Report bugs there if the ESP is crashing or a feature is not working as expected.
  - esphomedocs [https://github.com/OttoWinter/esphomedocs] - Report bugs there if the documentation is wrong/outdated.
- Provide as many details as possible. Paste logs, configuration sample and code into the backticks (```). Do not delete any text from this template!
-->

**Operating environment (Hass.io/Docker/pip/etc.):**

pip

**ESP (ESP32/ESP8266/Board/Sonoff):**

ESP8266
Wemos D1 mini

**Affected component:**

https://esphomelib.com/esphomeyaml/components/display/lcd_gpio.html#display-lcd-lambda

**Description of problem:**

Can't compile `it.strftime("It is %H:%M on %d.%m.%Y");`

**Problem-relevant YAML-configuration entries:**
```yaml
display:
  - platform: lcd_pcf8574
    dimensions: 18x2
    address: 0x27
    lambda: |-
      it.strftime("It is %H:%M on %d.%m.%Y");
```

**Traceback (if applicable):**
<!--
Please copy the traceback here if compilation is failing. If possible, also connect to the ESP and copy its logs into the backticks.
-->
```
src/main.cpp: In lambda function:
src/main.cpp:31:44: error: no matching function for call to 'esphomelib::display::LCDDisplay::strftime(const char [24])'
it.strftime("It is %H:%M on %d.%m.%Y");
^
src/main.cpp:31:44: note: candidates are:
In file included from .piolibdeps/esphomelib/src/esphomelib/application.h:39:0,
from src/main.cpp:2:
.piolibdeps/esphomelib/src/esphomelib/display/lcd_display.h:47:8: note: void esphomelib::display::LCDDisplay::strftime(uint8_t, uint8_t, const char*, esphomelib::time::EsphomelibTime)
void strftime(uint8_t column, uint8_t row, const char *format, time::EsphomelibTime time) __attribute__ ((format (strftime, 4, 0)));
^
.piolibdeps/esphomelib/src/esphomelib/display/lcd_display.h:47:8: note:   candidate expects 4 arguments, 1 provided
In file included from .piolibdeps/esphomelib/src/esphomelib/application.h:39:0,
from src/main.cpp:2:
.piolibdeps/esphomelib/src/esphomelib/display/lcd_display.h:49:8: note: void esphomelib::display::LCDDisplay::strftime(const char*, esphomelib::time::EsphomelibTime)
void strftime(const char *format, time::EsphomelibTime time) __attribute__ ((format (strftime, 2, 0)));
^
.piolibdeps/esphomelib/src/esphomelib/display/lcd_display.h:49:8: note:   candidate expects 2 arguments, 1 provided
*** [.pioenvs/d1_lcd/src/main.cpp.o] Error 1

```

**Additional information:**

using `it.print(2, 0, "Hello World!");` prints Hello World like expected on the display

--- a/esphomeyaml/components/display/lcd_gpio.rst
+++ b/esphomeyaml/components/display/lcd_gpio.rst
@@ -92,8 +92,13 @@ by default which means the character at the top left.
           // Result: "Sensor value:       42"
 
           // Print the current time
-          it.strftime("It is %H:%M on %d.%m.%Y");
+          it.strftime("It is %H:%M on %d.%m.%Y", id(my_time).now());
           // Result for 10:06 on august 21st 2018 -> "It is 10:06 on 21.08.2018"
+
+    # (Optional) For displaying time:
+    time:
+    - platform: sntp
+      id: my_time
 
 .. note::
 

